docs/faq/groups.md

```diff
@@ -28,38 +28,40 @@ Temporary Sync Group players are configured via the [Player List](ui.md#player-l
+These are permanent group players that are configured via the MA settings. These should be used when the make up of the group will rarely change and it is desired that the players will typically all continuously play music. A player cannot be removed from the group if the group is playing although it can be muted (or volume reduced to 0). If there is a need to add or remove additional players then enable the dynamic members option or switch to using a manual sync group. At power on/off of the syncgroup the original set of members will be restored.
 
 ![image](../assets/screenshots/syncgroup.png)
 
-After selecting ADD GROUP PLAYER as shown above, the group configuration screen will appear where the group type must be selected. At least one player must be added to the group before the group can be saved. It is recommended that a custom name be used as the automatically generated name is unique but not memorable.
+After selecting ADD GROUP PLAYER as shown above, the group configuration screen will appear where the group type must be selected. It is recommended that a custom name be used as the automatically generated name is unique but not memorable. The order that the players are selected when the group is created will be the order that the players are shown when viewing the group in the [Player List](../ui.md#player-list).
 
 !!! tip
     Remember Chromecast devices must be grouped in the Google Home app for perfect sync
 
 ### Features
 
-- The native player types shown (i.e. not the Universal Group option) support perfect sync (this is a requirement to be shown in the list)
+- The native player types shown (i.e. not the Universal Group option) support perfect sync
 - In addition to playing in sync the group will hold the queue regardless of which player(s) in the group are powered off
   
 ### Known Issues / Notes
 
 - As mentioned above the group player holds the queue rather than any individual player. However, if the group leader is lost then playback will stop but the queue can be restarted
 - The group will not power on if one of the child players is synced to another group
 - If, in the group settings, the player friendly name has been replaced by an unusual ID then likely the player was seen by the MA logic as unavailable. Check the individual player status
+- When adding a player to an already playing group, Airplay, Snapcast and Sonos will keep playing but all other types will pause for a short period
```

Comment:
```suggestion
- When adding a player to an already playing group, some player types will pause for a very brief period to resync.
```

Reply:
The reason I added a list is because people ask if the pause they experienced is normal or a bug? This answers the question for them?